internal/provider/resource_apply_module.go

```diff
@@ -191,6 +191,49 @@ func (t *applyModuleResource) Schema(_ context.Context, _ resource.SchemaRequest
 					},
 				},
 			},
+			"speculative": schema.BoolAttribute{
+				MarkdownDescription: "Whether the run will be speculative, default is false.",
+				Description:         "Whether the run will be speculative, default is false.",
+				Optional:            true,
+				Computed:            true, // When not passed it, it needs to be set by Create.
```

Comment:
`speculative` should not need to be computed. It can just have a default value of false.